lib/src/editor/editor_component/service/selection/mobile_selection_service.dart

```diff
@@ -401,6 +449,14 @@ class _MobileSelectionServiceWidgetState
     final selectable = node.selectable;
     final cursorRect = selectable?.getCursorRectInPosition(position);
     if (selectable != null && cursorRect != null) {
+      Log.selection
+          .debug("reveal:${_getOffsetToRevealCaret(cursorRect).offset}");
+      editorState.service.scrollService?.scrollController.animateTo(
+        _getOffsetToRevealCaret(cursorRect).offset,
+        duration: const Duration(milliseconds: 100),
+        curve: Curves.easeIn,
+      );
```

Comment:
If we do need the Log, then at least:

```suggestion
      final caretOffset = _getOffsetToRevealCaret(cursorRect).offset;
      Log.selection.debug("reveal: $caretOffset");
      editorState.service.scrollService?.scrollController.animateTo(
        caretOffset,
        duration: const Duration(milliseconds: 100),
        curve: Curves.easeIn,
      );
```

Reply:
ok